Common/Models/Parameter.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models
 {
     public class Parameter
     {
+        /// <summary>
+        /// Serialization deserialization constructor.
+        /// </summary>
+        [JsonConstructor]
```

Comment:
it's used in deserialization. It has another constructor with params. The deserializer tries using that if the empty one is not defined.